### PR TITLE
chore: Tweak UI to adapt to sidebar blur

### PR DESCRIPTION
**Pull request by FeiWang1119**
tweak UI to adapt to sidebar blur

Log:

### CODE REVIEW

**Review by Decodetalkers — DISMISSED** [review]

File: src/frame/listitemdelegate.cpp [hunk 1]
```diff
@@ -89,21 +92,27 @@ void ListItemDelegate::paint(QPainter *painter, const QStyleOptionViewItem &opti
...
 
     drawDecoration(painter, opt, decorationRect);
 
     drawDisplay(style, painter, opt, displayRect);
 
     if (isIconMode || isBeginning) {
-        painter->setPen(opt.palette.color(QPalette::Disabled, QPalette::Text));
+        painter->save();
+        painter->setPen(opt.palette.color(cg, role));
+        painter->setOpacity(0.5);
```
Comment (kegechen) [inline src/frame/listitemdelegate.cpp:110]: 0.7

**Review by kegechen — CHANGES_REQUESTED** [review]

**Review by kegechen — DISMISSED** [review]

File: src/frame/listitemdelegate.cpp (hunk 1 above)
Comment (FeiWang1119) [reply src/frame/listitemdelegate.cpp]: done

**Review by kegechen — APPROVED** [review]

**Pull request merged**